### ISSUE TITLE
.arc-env mentioned only once, not documented

### ISSUE THREAD

**Issue by lpsinger** [architect/arc.codes#488]
**Describe the issue**
According to https://arc.codes/docs/en/reference/cli/sandbox:

> `ARC_LOCAL`- If present and used in conjunction with `NODE_ENV=staging|production`, emulates live staging or production environment
> ...
>
> * Uses your local `.arc-env` file’s `@staging` or `@production` environment variables
>
> ...

This is the only mention that I can find of the `.arc-env` file. I see no other documentation about the `.arc-env` file, and no explanation of its format or contents. I suspect that this is actually supposed to refer to `preferences.arc`.

**Steps to reproduce**
N/A, this is a documentation bug.

**Expected behavior**
N/A, this is a documentation bug.

**Screenshots**
N/A, this is a documentation bug.

**Comment by tbeseda** [comment]
Good find @lpsinger 
We need to focus the docs on the preferred way to use environment variables so they work locally and when deployed.
@ryanblock, I know some of this has shifted lately. What's the canonical Arc way here? and is .arc-env useful? It seems preferences.arc is more common in our projects?
thanks

**Comment by ryanblock** [comment]
`.arc-env` is deprecated and the last remaining support will be removed in Arc 10 – we should remove all references. It was superseded by `pref[erence]s.arc`.
